### PR TITLE
Feat/loading

### PR DESCRIPTION
**Pull request by abj10**
## What?

This PR integrates the `LoadingScreen` into the navigation flow and updates `MainActivity`, `NavigationActions`, and `SignIn` components.

## Why?

To provide smoother transitions between the login screen, loading screen, and dashboard

## How?

- Updated `MainActivity` to include the new `LoadingScreen` in the navigation flow.
- Modified `SignInScreen` to route through `LoadingScreen` after authentication.
- Adjusted `NavigationActions` to support navigation to the `LoadingScreen`.
- Added `LoadingScreen` to facilitate transitions between login and dashboard.
- Introduced `PermissionDeniedScreen` to handle camera permission requests.
- Updated branding with new logos (`eduverse_logo.png`, `eduverse_logo_alone.png`).

## Type of change

- [x] New feature (integrated `LoadingScreen` and updated navigation flow).
- [x] UX improvements (smoother transitions between screens and new logos).

## Checklist

- [x] Code follows the project's style guidelines.
- [x] Verified that navigation transitions work as expected.
- [x] Tested the Loading Screen.
- [x] Updated branding with new logos.

### CODE REVIEW

**Review by MathieuxC — CHANGES_REQUESTED** [review]
The changes look good to me, but you should write test.

File: app/src/main/java/com/github/se/eduverse/ui/authentification/Loading.kt
Comment (Fayroth) [inline app/src/main/java/com/github/se/eduverse/ui/authentification/Loading.kt:1]: I dont think you should use the auth.currentID to bypass the connection step. It always lead to the AUTH screen everytime.

**Review by Fayroth — APPROVED** [review]
You could also add a loading circle for ui purposes but otherwise its all good, looks good to me.

**Review by Fayroth — APPROVED** [review]
LGTM

**Review by MathieuxC — APPROVED** [review]

**Pull request merged**